vendor/k8s.io/kubernetes/pkg/kubectl/plugins/plugins.go

```diff
@@ -71,3 +81,33 @@ func (p Plugin) IsValid() bool {
 
 // Plugins is a list of plugins.
 type Plugins []*Plugin
+
+// Flag describes a single flag supported by a given plugin.
+type Flag struct {
```

Comment:
I couldn't think of any use case where there would be different implementations of these guys, so I don't think it's worth at the moment.  ;)